Module2/springhometask/src/test/java/com/epam/spring/TestTickets.java

```diff
@@ -25,10 +25,10 @@ public void init(){
     }
 
     private void initListOfTickets() {
-        ticketList.add(TicketEntity.createNewTicket(0, 0, 1,1, Ticket.Category.STANDARD));
-        ticketList.add(TicketEntity.createNewTicket(1, 0, 0,10, Ticket.Category.PREMIUM));
-        ticketList.add(TicketEntity.createNewTicket(2, 1, 1,13, Ticket.Category.BAR));
-        ticketList.add(TicketEntity.createNewTicket(3, 1, 0,18, Ticket.Category.BAR));
+        ticketList.add(new TicketFactory().createTicket(0, 0, 1,1, Ticket.Category.STANDARD));
```

Comment:
multiple creation of exactly same object through new() - not a good idea. In real app it leads to memory leaks and bugs

Reply:
ok, I changed it in creating object of TicketFactory only once.